src/pages/info/LocationInfo.tsx

```diff
@@ -66,7 +68,7 @@ export default function LocationInfo({ item }: LocationInfoProps): JSX.Element {
               next={(): Character[] => characters}
               hasMore
               loader={<h4>{t('loading')}</h4>}
-              height={550}
+              height={height / 2.5}
```

Comment:
2.5?

Reply:
should I move it to consts or what ?

Reply:
it would be great) f.e object screenSizeProportions